src/routes/resources/resources.handlers.ts

```
@@ -149,6 +174,28 @@ export const patch: AppRouteHandler<PatchRoute> = async (c) => {
       HttpStatusCodes.NOT_FOUND
     );
   }
+  if (resource.image && isValidImageType(resource.image)) {
+    return c.json(
+      { message: "Invalid image type", success: false },
+      HttpStatusCodes.BAD_REQUEST
+    );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)